api/src/team/domain/usecases/update-membership-last-accessed-at.usecase.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Update the last accessed at date of a membership
+ * @param {Object}
+ * @param {string} userId - The ID of the user
+ * @param {string} organizationId - The ID of the organization
+ * @param {MembershipRepository} membershipRepository - The repository for memberships
+ */
+const updateMembershipLastAccessedAt = async function ({ userId, organizationId, membershipRepository }) {
```

Comment:
1. Suppression des paraphrases inutiles _« Update the last accessed at date of a membership »_, etc.
2. Correction de la déclaration JSDoc des propriétés cf. https://jsdoc.app/tags-param

```suggestion
/**
 * @param {Object} params
 * @param {string} params.userId
 * @param {string} params.organizationId
 * @param {MembershipRepository} params.membershipRepository
 */
const updateMembershipLastAccessedAt = async function ({ userId, organizationId, membershipRepository }) {
```